api2-war/src/main/java/eu/europeana/api2/v2/web/controller/SearchController.java

```diff
@@ -969,23 +1027,21 @@ public ModelAndView fieldTripRss(
     }
 
     /**
-     * Temporary method to facilitate debugging the facet tags
+     * Method to find all encoded facets in tags
      *
      * @return the JSON response
      */
     @SwaggerIgnore
-    @GetMapping(value = "/api/v2/tagdecoder.json", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ModelAndView searchJson(
-            @RequestParam(value = "tag") String tag) {
+    @GetMapping(value = {"/v2/decodetags.json", "/v2/tagdecoder.json"},
+                produces = org.springframework.http.MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Endpoints should start with /api (for consistency)

Reply:
Done.